### PR TITLE
show more upcoming stops

### PR DESCRIPTION
**Pull request by Mxdcq**

### CODE REVIEW

File: app/css/style.css [hunk 1]
```diff
@@ -61,6 +67,12 @@ md-card[md-ink-ripple] {
   font-weight: bold;
 }
 
+/*md-sidenav, md-sidenav.md-locked-open, md-sidenav.md-closed.md-locked-open-add-active {*/
```
Comment (antti-TIEA4) [inline app/css/style.css]: these commented lines are not needed

File: app/css/style.css [hunk 2]
```diff
@@ -31,9 +35,11 @@ md-card[md-ink-ripple] {
 .event-list {
   background-color: #FFCA28;
   border: 1px solid #000;
-  height: 150px;
+  height: 100px;
   text-overflow: ellipsis;
   padding: 1%;
+  margin: 0;
+  width:100%;
```
Comment (antti-TIEA4) [inline app/css/style.css]: is this line necessary?

File: app/index.html [hunk 3]
```diff
@@ -34,42 +34,44 @@
 
 <body ng-app="upcomingStops" ng-if='$root.initialized' layout="column">
 
-  <div ng-controller="EventController as ec" layout-fill>
+<div ng-controller="EventController as ec" layout-fill>
```
Comment (antti-TIEA4) [inline app/index.html]: the indentation of index.html is not right

File: app/index.html [hunk 4]
```diff
@@ -34,42 +34,44 @@
...
+    <md-card ng-cloak class="event-tile show-more" flex layout-fill layout-align="center center">
+      Show more upcoming stops
+    </md-card>
+  </md-content>
 
-    <div ng-if=$root.fullscreen flex layout-fill layout="row">
-      <md-sidenav md-is-open="true" md-component-id="left" ng-click="ec.toggleEventList()" class="md-sidenav-left md-whiteframe-z3" layout="column">
-        <md-card ng-repeat="event in ec.events | orderBy : 'distanceToUser'" ng-cloak flex layout-fill md-ink-ripple ng-click="ec.selectEvent(event)" ng-class="{'selected' : event === event.selectedEvent }" class="event-list disable-text-selection">
+  <div ng-if=$root.fullscreen flex layout-fill layout="row">
+    <md-sidenav md-is-open="fasle" md-component-id="left" ng-click="ec.toggleEventList()" class="md-sidenav-left md-whiteframe-z3">
```
Comment (antti-TIEA4) [inline app/index.html]: no need to change this, `md-is-open` should remain `true`

File: app/css/style.css (hunk 2 above)
Comment (Mxdcq) [inline app/css/style.css:42]: Yes, it is necessary.

File: app/css/style.css (hunk 1 above)
Comment (Mxdcq) [inline app/css/style.css]: Deleted

File: app/index.html (hunk 3 above)
Comment (Mxdcq) [inline app/index.html]: done

File: app/index.html (hunk 4 above)
Comment (Mxdcq) [inline app/index.html]: done

File: app/index.html [hunk 5]
```diff
@@ -58,18 +61,18 @@
       <md-content ng-cloak flex id="map">
         <apt-map ng-style='{"width": "100%", "height": "100%"}'></apt-map>
         <md-button class="md-icon-button md-raised md-hue-2 event-list-button" hide-gt-sm ng-click="ec.toggleEventList()" aria-label='Open Event List'>
-          <!--<md-icon md-svg-icon="img/icons/menu.svg" ></md-icon>-->
+          <!--<md-icon md-svg-icon="img/iconsis/menu.svg" ></md-icon>-->
```
Comment (antti-TIEA4) [inline app/index.html]: Hmm... There's no such directory as `img/iconsis/`
When we switch back to SVG icons, this would introduce a bug.

**Comment by antti-TIEA4** [comment]
Thanks @Mxdcq, in master :+1:

**Comment by Mxdcq** [comment]
You are welcome.

**Pull request merged**